src/app/components/phase-banner/phase-banner.component.ts

```diff
@@ -18,7 +19,12 @@ export class PhaseBannerComponent implements OnInit {
   }
 
   public toggleLanguage(lang: RpxLanguage): void {
+    this.noBanner = (lang === 'cy');
     this.langService.language = lang;
   }
 
+  public closeBanner() {
+    this.noBanner = true;
```

Comment:
do we need to store this in sessionStorage or localStorage? Could you confirm with Adeel whether need to remember this selection for next time logon?